Gems/Atom/RHI/Code/Include/Atom/RHI/XRRenderingInterface.h

```diff
@@ -15,6 +15,10 @@
 
 namespace AZ::RHI
 {
+    class Image;
+    //! Key for the desired foveated rendering level
+    inline constexpr const char* XRFoveatedLevelKey = "/O3DE/Atom/RHI/XRFoveatedLevel";
```

Comment:
We've put other XR parameters under `/O3DE/Atom/OpenXR`. Strictly speaking this code is under Atom RHI interface and I see why it's been put there, but I wonder if from a practical standpoint it makes more sense to put it under `/O3DE/Atom/OpenXR` to keep them together.

So right now a typical OpenXR.setreg file would look like this:

````
{
    "O3DE": {
        "Atom": {
            "RHI": {
                "XRFoveatedLevel": 2
            },
            "OpenXR": {
                "Enable": true
            }
        }
    }
}
````

Versus like this if changed to `/O3DE/Atom/OpenXR`:

````
{
    "O3DE": {
        "Atom": {
            "OpenXR": {
                "Enable": true,
                "FoveatedLevel": 2
            }
        }
    }
}
````

I think I inclined more toward the second one. Let me know what you think.